kernel/Makefile.dist.install

```diff
@@ -0,0 +1,19 @@
+# Installation variables
```

Comment:
Could you add a copyright header for this file?

Reply:
Done